eval/command.go

```diff
@@ -134,6 +135,19 @@ func env(cmd string, args []value.Value) (value.Value, error) {
 	}, nil
 }
 
+// UuidCmd implements the uuid command for generating UUIDs.
+var UuidCmd = &Command{
+	Name: "uuid",
+	Argc: -1,
+	Func: uuidFn,
```

Comment:
The implementation of the command is pretty simple, I think the callback could be inlined here, similar to what we do for the different method commands, see `GetCmd`, `PostCmd`, etc. for what I mean.